fleet/lib/rook-ceph-cluster/overlays/manke/values.yaml

```diff
@@ -69,12 +69,19 @@ cephBlockPools:
       quotas:
         maxSize: 15Ti
       enableRBDStats: true
+      parameters:
+        nodelete: "true"
+        nosizechange: "true"
+        pg_autoscale_mode: "off"
+        pg_num: "512"
     storageClass:
       name: rook-ceph-block
       enabled: true
       isDefault: true
       reclaimPolicy: Delete
       allowVolumeExpansion: true
+      mountOptions:
+        - discard
```

Comment:
Nice catch.

Reply:
thanks! i based on the pillan branch but looked at the current config on elqui too